kubernetes-manifests/ocagent.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oc-agent-conf
+  labels:
+    app: opencensus
+    component: oc-agent-conf
+data:
+  oc-agent-config: |
+    receivers:
+      opencensus:
+        address: "0.0.0.0:55678"
+      jaeger: {}
+      zipkin: {}
+    exporters:
+      stackdriver:
+        # TODO: find a different way to specify project-id
```

Comment:
can we make oc-agent determine the project-id from the metadata server, just like how all stackdriver services do?

Ideally we should not have a projectID configuration in yaml. Currently the demo app is very easy to deploy because we don't require users to change anything, and we should try to keep it that way.

Reply:
I had same concern. It should be possible but let me check with @songy23.

Reply:
https://github.com/census-instrumentation/opencensus-service/pull/526 should fix it, you can leave this field blank and Agent will use the default project ID.